File: app/src/main/java/com/stepyen/xlearn/fragment/basics/PaintFragment.java
package com.stepyen.xlearn.fragment.basics;

import com.stepyen.xlearn.R;
import com.stepyen.xlearn.base.BaseTestFragment;
import com.xuexiang.xpage.annotation.Page;

/**
 * date：2019/7/5
 * author：stepyen
 * description：
 */
@Page(name = "Paint",extra = R.drawable.ic_widget_imageview)
public class PaintFragment extends BaseTestFragment {
    @Override
    public void initLayoutView() {
        addView(R.layout.fragmen_paint);
    }

    @Override
    protected void initViews() {

    }


}

File: app/src/main/java/com/stepyen/xlearn/fragment/basics/PaintFragment.kt
package com.stepyen.xlearn.fragment.basics

import com.stepyen.xlearn.R
import com.stepyen.xlearn.base.BaseTestFragment
import com.xuexiang.xpage.annotation.Page

/**
 * date：2019/7/5
 * author：stepyen
 * description：
 */
@Page(name = "Paint", extra = R.drawable.ic_widget_imageview)
class PaintFragment : BaseTestFragment() {
    override fun initLayoutView() {
        addView(R.layout.fragmen_paint)
    }

    override fun initViews() {

    }


}